assets/training/model_management/components/download_model/spec.yaml

```diff
@@ -1,7 +1,7 @@
 $schema: https://azuremlschemas.azureedge.net/latest/commandComponent.schema.json
 
 name: download_model
-version: 0.0.10
+version: 0.0.11
```

Comment:
Could you please take a pull from main? Latest version in 11 only.

Reply:
Ack